io.github.gitahead.GitAhead.json

```diff
@@ -0,0 +1,120 @@
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "gitahead",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    /*for git repos that require ssh keys*/
+    "--socket=ssh-auth",
+    "--share=network",
+    "--socket=pcsc",
+    /*we want access to all files under home*/
+    "--filesystem=home",
+    /*we use the keyring to store credentials*/
+    "--filesystem=xdg-run/keyring",
+    "--talk-name=org.freedesktop.secrets",
+    "--talk-name=org.freedesktop.Notifications"
+  ],
+  "modules": [
+    "shared-modules/libsecret/libsecret.json",
+    "shared-modules/intltool/intltool-0.51.json",
+    {
+      "name": "git",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 $(which git) /app/bin/",
+        "install -Dm0755 $(which git-clang-format) /app/bin/",
+        "install -Dm0755 $(which git-cvsserver) /app/bin/",
+        "install -Dm0755 $(which git-receive-pack) /app/bin/",
+        "install -Dm0755 $(which git-shell) /app/bin/",
+        "install -Dm0755 $(which git-upload-archive) /app/bin/",
+        "install -Dm0755 $(which git-upload-pack) /app/bin/"
+      ]
+    },
+    {
+      "name": "graphviz",
+      "cleanup": [
+        "/share/man",
+        "/share/graphviz/doc"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.com/graphviz/graphviz/-/archive/2.47.0/graphviz-2.47.0.tar.gz",
+          "sha256": "4b7f61c7c04f39b67b2c87977633d61d9f72ee6f93ff4376acdce140128431d4"
+        }
+      ]
+    },
+    {
+      "name": "libgnome-keyring",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--enable-vala=no",
+        "--enable-introspection=no"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download-fallback.gnome.org/sources/libgnome-keyring/3.12/libgnome-keyring-3.12.0.tar.xz",
+          "sha256": "c4c178fbb05f72acc484d22ddb0568f7532c409b0a13e06513ff54b91e947783"
+        }
+      ],
+      "cleanup":[
+        "/share/gtk-doc"
+      ]
+    },
+    {
+      "name": "gitahead",
+      "buildsystem": "cmake-ninja",
+      "builddir": true,
+      "build-options": {
+        "prefix": "/app/gitahead"
+      },
+      "cleanup": [
+        "/gitahead/*.so*",
+        "/gitahead/include",
+        "/gitahead/lib"
+      ],
+      "post-install": [
+        "install -Dm755 ../gitahead.sh /app/bin/gitahead",
+        "install -Dm644 /app/gitahead/Resources/GitAhead.iconset/icon_512x512.png /app/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png",
+        "install -Dm644 /app/gitahead/Resources/GitAhead.iconset/icon_128x128.png /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png",
+        "install -Dm644 /app/gitahead/Resources/GitAhead.iconset/icon_256x256.png /app/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png",
```

Comment:
copying the 256x256 icon to the 512x512?  Looks like a typo.

Reply:
ah thanks for pointing it out, I'll fix it